portfolio/pom.xml

```diff
@@ -34,7 +34,6 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <!-- TODO: set project ID. -->
           <deploy.projectId>yitaowang-pod-step-2020</deploy.projectId>
```

Comment:
I think I missed this in the last review. You should set it to your personal project instead of the team one. I believe it's sroblesher-step-2020

Reply:
Got it! Thank you